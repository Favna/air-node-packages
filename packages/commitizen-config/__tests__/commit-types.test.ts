import commitTypes from '../src/commit-types';

describe('Commit Types Snapshot', () => {
  test('should match snapshot', () => {
    expect(commitTypes).toMatchSnapshot();
  });
});
