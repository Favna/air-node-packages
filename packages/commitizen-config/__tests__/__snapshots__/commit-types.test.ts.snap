// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commit Types Snapshot should match snapshot 1`] = `
Object {
  "types": Object {
    "build": Object {
      "description": "For commits that change the build configuration",
      "title": "Builds",
    },
    "chore": Object {
      "description": "For chore commits that do not change code or documentation",
      "title": "Chores",
    },
    "ci": Object {
      "description": "For commits that change the CI/CD flow",
      "title": "Continuous Integrations / Continuous Deployment",
    },
    "dependencies": Object {
      "description": "For commits that change or upgrade dependencies",
      "title": "Dependencies",
    },
    "docs": Object {
      "description": "For commits that change documentation",
      "title": "Documentation",
    },
    "feat": Object {
      "description": "For commits that change the build configuration",
      "title": "Features",
    },
    "fix": Object {
      "description": "For commits that fix bugs",
      "title": "Bug Fixes",
    },
    "misc": Object {
      "description": "For commits that do not match any of the other types",
      "title": "Miscellaneous",
    },
    "refactor": Object {
      "description": "For commits that refactor existing code",
      "title": "Code Refactoring",
    },
    "revert": Object {
      "description": "For commits that revert previous commits",
      "title": "Reverts",
    },
    "style": Object {
      "description": "For commits that exclusively change the style of the code, such as indents, linting, and auto-formatting",
      "title": "Styling",
    },
    "test": Object {
      "description": "For commits that change test code",
      "title": "Tests",
    },
    "wip": Object {
      "description": "For commits that add work-in-progress code, should never be used on the master branch but can be used on non-master branches",
      "title": "WIP",
    },
  },
}
`;
